spec/tom_queue/deferred_work/deferred_work_manager_integration_spec.rb

```diff
@@ -19,13 +19,15 @@ def notify(exception)
 
   let!(:file) { Tempfile.new("exception") }
   let!(:pid) do
-    fork do
+    forked_process = TestForkedProcess.new do
       TomQueue.bunny = Bunny.new(TEST_AMQP_CONFIG)
       TomQueue.bunny.start
       TomQueue.exception_reporter = exception_reporter
       manager = TomQueue::DeferredWorkManager.new(TomQueue.default_prefix)
       manager.start
     end
+    forked_process.start
+    forked_process.pid
```

Comment:
This means the orphan processes get cleaned up, but still need to figure out:

a) Why the tests fail
b) Why the processes are being orphaned